tests/org.corpus_tools.hexatomic.it.tests/src/main/java/org/corpus_tools/hexatomic/it/tests/TestGridEditor.java

```diff
@@ -545,8 +546,10 @@ void testPopupMenuInvisibleOnSelectedTokenText() {
     SWTNatTableBot tableBot = new SWTNatTableBot();
     SWTBotNatTable table = tableBot.nattable();
     table.click(1, 1);
-    assertThrows(WidgetNotFoundException.class,
-        () -> table.contextMenu(1, 1).contextMenu("Delete cell(s)"));
+    SWTBotRootMenu contextMenu = table.contextMenu(1, 1);
+    // No context menu should exist 
+    assertTrue(contextMenu.menuItems().isEmpty());
+    assertThrows(WidgetNotFoundException.class, () -> contextMenu.contextMenu());
```

Comment:
I'm just applying the suggestion from the static code analysis, which also seems more readable
```suggestion
    assertThrows(WidgetNotFoundException.class, contextMenu::contextMenu);
```